phpBB/includes/acp/acp_language.php

```diff
@@ -229,7 +229,20 @@ function main($id, $mode)
 
 					$phpbb_log->add('admin', $user->data['user_id'], $user->ip, 'LOG_LANGUAGE_PACK_DELETED', false, array($row['lang_english_name']));
 
-					trigger_error(sprintf($user->lang['LANGUAGE_PACK_DELETED'], $row['lang_english_name']) . adm_back_link($this->u_action));
+					$delete_message = sprintf($user->lang['LANGUAGE_PACK_DELETED'], $row['lang_english_name']);
+					$lang_iso = $row['lang_iso'];
```

Comment:
I think that modifying lang_iso variable here has no effect

Reply:
yes, only to inform which language deleted

Reply:
Are you sure?